### ISSUE TITLE
Client crash

### ISSUE THREAD

**Issue by xenohedron** [magefree/mage#12675]
> [xcb] Unknown sequence number while processing queue
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
java: ../../src/xcb_io.c:278: poll_for_event: Assertion `!xcb_xlib_threads_sequence_lost' failed.

**Comment by JayDi85** [comment]
It’s incompatible libx11 v1.8 library on linux systems. Must be upgraded to newer or downgraded to older version. Details and possible versions can be found here: https://github.com/ppy/osu/discussions/21474#discussioncomment-4436812

**Comment by xenohedron** [comment]
Interesting. I'm on mint, not arch, and as far as I can tell that is not the library version I have.

Seems to be resolved by a fresh download from xmage.today ...

**Comment by xenohedron** [comment]
Nope, still getting this crash, seems to be when a match starts but xmage is in the background
